.github/workflows/ci.yml

```diff
@@ -75,25 +88,33 @@ jobs:
 
   build:
     needs: lint
-    name: Build GTK Binaries
+    name: Build GTK${{ matrix.gtk-version }} Binaries
     runs-on: windows-latest
     timeout-minutes: 75
     if: "!contains(github.event.head_commit.message, 'skip ci')"
+    outputs:
+      targz: gvsbuild-${{ steps.meta.outputs.version }}.tar.gz
+      wheel: gvsbuild-${{ steps.meta.outputs.version }}-py3-none-any.whl
     strategy:
       matrix:
         gtk-version: ['3', '4']
 
     steps:
+      - name: Harden Runner
+        uses: step-security/harden-runner@55d479fb1c5bcad5a4f9099a5d9f37c8857b2845 # v2.4.1
+        with:
+          egress-policy: audit
+
       - name: Checkout
-        uses: actions/checkout@v3.5.3
+        uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
```

Comment:
will these hard coded hashes get updated by the automation or are we stuck with them until we manually update?

Reply:
Hey @pbor, Dependabot automatically updates the hashes

Reply:
Ok, then I am ok with this